grammar.js

```diff
@@ -31,7 +31,7 @@ module.exports = grammar({
     ),
 
     setting: $ => seq(
-      alias(/[^#=\s\[]+/, $.setting_name),
+      alias(/[^;|#=\s\[]+/, $.setting_name),
```

Comment:
I assume the `|` was not intentional. In a `[...]` atom that would be taken literally, it's not an "OR".

Reply:
you're right